lib/configParser.js

```diff
@@ -33,6 +33,7 @@ var ConfigParser = function() {
       defaultTimeoutInterval: (30 * 1000)
     },
     seleniumArgs: [],
+    sessionId: null,
```

Comment:
Let's rename to `seleniumSessionId` - to err on the side of being too verbose but obvious what this belongs to.